CSVSource/src/test/java/io/vantiq/extsrc/CSVSource/TestCSVBase.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Vantiq, Inc.
+ *
+ * All rights reserved.
+ * 
+ * SPDX: MIT
+ */
+
+package io.vantiq.extsrc.CSVSource;
+
+import org.junit.BeforeClass;
+
+public class TestCSVBase {
+    static String testVantiqServer;
+    static String testAuthToken;
+
+    static String testDelimiter;
+    static String testFileFolderPath;
+    static String testFilePrefix;
+    static String testFileExtension;
+    static String testFullFilePath ; 
+    static int testMaxLinesInEvent;
+    
+    @BeforeClass
+    public static void getProps() {
+        testAuthToken = System.getProperty("TestAuthToken", null);
+        testVantiqServer = System.getProperty("TestVantiqServer", null);
+
+        testFileFolderPath = System.getProperty("EntFileFolderPath", "c:/tmp/csvTest/in");
```

Comment:
For things with file paths that you expect, you should remove these defaults.  They are very _your machine_ and _windows_ specific.

I've updated the build.gradle file to take these values from the gradle.properties file & convert them to java system properties.  That's the proper way to do this.

Also, by giving them defaults, all the `assumeTrue()` calls in the code don't keep the tests from running.  What's supposed to happen is that if these values aren't specified, the associated tests don't run -- they just show up as skipped. But they don't fail which is what currently happens.  I've run the tests -- without overriding these values, various tests fail.  

We want it to be the case that if someone downloads & compiles the extension, it should pass its tests. This is also why these settings need to be described in the README.

Reply:
add support for assume for check path not exists .